packages/one-app-bundler/__tests__/bin/serve-module.spec.js

```diff
@@ -68,6 +80,19 @@ describe('serve-module', () => {
     expect(fs.mkdirSync.mock.calls[1][0]).toEqual('/mocked/static/modules/my-module-name');
   });
 
+  it('should create a directory for the module without legacy', () => {
```

Comment:
Ended up removing this test because the check we care about happens in a test further down. 
`adds to the existing module map without legacy when disableDevelopmentLegacyBundle is true`